goss/goss.yaml

```diff
@@ -24,6 +24,11 @@ command:
     stdout:
       - 21-beta
     stderr: []
+  playwright --version:
```

Comment:
```suggestion
  playwright --version:
  /home/jenkins/.asdf/shims/playwright --version:
```

Reply:
@hainenber  I probably sent you in the wrong way here, the problem is not the path but the context, let me try a little more and I will show/explain this

Reply:
FYI
```
jenkins@ac02732a066b:/tmp$ cat /home/jenkins/.asdf/shims/playwright 
#!/usr/bin/env bash
# asdf-plugin: nodejs 18.17.0
exec /home/jenkins/.asdf/bin/asdf exec "playwright" "$@" # asdf_allow: ' asdf '
```

Reply:
The `exec` command to check for `playwright` version is now ran under `jenkins` user. Successfully built and tested on my local machine

<img width="1068" alt="Screenshot 2023-09-21 at 23 43 52" src="https://github.com/jenkins-infra/packer-images/assets/41283691/f77c952a-22e9-4b9b-99c8-bdc4e2db7aff">

Reply:
The `exec` command to check for `playwright` version is now ran under `jenkins` user. Successfully built and tested on my local machine

<img width="1068" alt="Screenshot 2023-09-21 at 23 43 52" src="https://github.com/jenkins-infra/packer-images/assets/41283691/f77c952a-22e9-4b9b-99c8-bdc4e2db7aff">